benchmark/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.eclipse.rdf4j</groupId>
+    <artifactId>rdf4j-benchmark</artifactId>
+    <version>2.2-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
```

Comment:
I think you're missing a parent section. If you make it a descendant of the root pom, it integrates better with the rest of the project.